api/envoy/extensions/clusters/dynamic_forward_proxy/v3/cluster.proto

```diff
@@ -32,4 +32,16 @@ message ClusterConfig {
   // in the :ref:`cluster's upstream_http_protocol_options
   // <envoy_v3_api_field_config.cluster.v3.Cluster.upstream_http_protocol_options>`
   bool allow_insecure_cluster_options = 2;
+
```

Comment:
Please add a `version_history.rst` entry for this feature.

Reply:
Oh! I didn't know that was a thing. Thanks. Done.

Reply:
Docs build looks broken, suggest generating locally and verifying build/output.